app/javascript/components/index.js

```diff
@@ -36,10 +36,11 @@ export const coreComponents = [
       fetchTargetComputeUrls: {
         rhevm:
           '/api/clusters?expand=resources' +
-          '&attributes=ext_management_system.emstype,v_parent_datacenter,ext_management_system.name' +
+          '&attributes=ext_management_system.emstype,v_parent_datacenter,ext_management_system.name,hosts' +
           '&filter[]=ext_management_system.emstype=rhevm',
-        openstack: '/api/cloud_tenants?expand=resources&attributes=ext_management_system.name'
-      }
+        openstack: '/api/cloud_tenants?expand=resources&attributes=ext_management_system.name,hosts'
```

Comment:
Oops, I had not considered this, `CloudTenants` do not have hosts

Reply:
Hmm. @michaelkro does that mean we need to make a special case for them with respect to this warning? or should they be exempt from it?  Currently they will just always show the warning if there are no hosts

Reply:
I think we need to find out if the concept of `transformation_host` is a thing in the OSP case

Reply:
I pushed a commit that removes it from that call, and also makes the warning icon only appear if there were hosts to speak of.